### PR TITLE
DDCNL-9626: update to scala 3

### PR DESCRIPTION
**Pull request by pascal22p**
- drop support of scala 2.12 and play 2.8
- added support of scala 3
- updated bootstrap to 9.5.0

### CODE REVIEW

**Review by grantrobson — APPROVED** [review]
lgtm

**Review by SandeepUmredkar — COMMENTED** [review]
println()

File: project/plugins.sbt [hunk 1]
```diff
@@ -4,14 +4,14 @@ resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("uk.gov.hmrc"   % "sbt-auto-build" % "3.22.0")
-addSbtPlugin("org.scoverage" % "sbt-scoverage"  % "2.0.11")
+addSbtPlugin("uk.gov.hmrc"   % "sbt-auto-build"     % "3.22.0")
 addSbtPlugin("uk.gov.hmrc"   % "sbt-distributables" % "2.5.0")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt"       % "2.5.2")
+addSbtPlugin("org.scoverage" % "sbt-scoverage"      % "2.2.1")
```
Comment (SandeepUmredkar) [inline project/plugins.sbt:10]: Latest is 2.2.2. If 2.2.1 is fine I assume 2.2.2 should be good as well

File: sca-wrapper-play-30/src/test/scala/views/ScaLayoutViewSpec.scala [hunk 2]
```diff
@@ -67,6 +67,7 @@ class ScaLayoutViewSpec extends ViewBaseSpec {
 
   "WrapperService layout" must {
     "return a Wrapper layout with default parameters in English" in {
+      println(createView().toString())
```
Comment (SandeepUmredkar) [inline sca-wrapper-play-30/src/test/scala/views/ScaLayoutViewSpec.scala]: println

**Pull request closed without merging**